mirrord/agent/src/steal/connection.rs

```diff
@@ -505,6 +505,15 @@ impl TcpConnectionStealer {
         client_id: ClientId,
         port: Port,
     ) -> Result<(), AgentError> {
+        // Remove all existing connections of the unsubscribing client on that port.
```

Comment:
when you stop listening (close the listening socket) it doesn't close all the established connections..

Reply:
Hey, sorry, just converted this PR into a draft to remove this part 😆 

Removing the connections will of course happen when the layer sees the connection is closed and send a `ConnectionUnsubscribe` to the agent. Don't know what I was thinking when I added these lines.

They also remove all the client's connections, no matter what port. 🤦‍♂️